api/database/migrations/20191121212344-change_hackathon_count.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.changeColumn("HackerProfiles", "hackathonCount", {
+      type: Sequelize.ENUM,
+      values: ["0", "1", "2", "3", "4", "5+"],
+      defaultValue: "0",
```

Comment:
lets let this be NULL and have that be the default value. this way we can differentiate between applications that were submitted before/after this change is introduced

otherwise, we'll get a bias towards 0